packages/themes/__tests__/mixins-test.js

```diff
@@ -31,6 +31,9 @@ describe('_mixins.scss', () => {
     const themeTests = Object.keys(themes).map(key => {
       const variable = `$carbon--theme--${key}`;
       const test = `
+        $feature-flags: (
```

Comment:
it seems setting the feature flag does not affect the test result, need to look more into this